.travis/download_html_validator

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/python3
```

Comment:
A nice to have, not must, a quick comment in this file describing the general overview of what is happening could potentially be useful.